test/test_ops.py

```diff
@@ -29,13 +29,14 @@ class TestOpInfo(TestCase):
     @onlyOnCPUAndCUDA
     @ops(op_db, dtypes=OpDTypes.unsupported)
     def test_unsupported_dtypes(self, device, dtype, op):
-        samples = op.sample_inputs(device, dtype)
-        if len(samples) == 0:
-            self.skipTest("Skipped! No sample inputs!")
-
-        # NOTE: only tests on first sample
-        sample = samples[0]
+        # sample_inputs can have a function for generating the input that doesn't work for specified dtype
```

Comment:
This is a really interesting point. The goal of this test is to verify that the function doesn't work on the given dtypes, but it makes sense that OpInfo implementers aren't designing sample input functions that support these dtypes!

This workaround seems OK for now, but please file an issue for this problem and add a comment citing the issue so we can revisit this later.